lib/skew.gd

```diff
@@ -27,30 +27,19 @@ BindGlobal("SkewBraceElmFamily", NewFamily("SkewBraceElmFamily", IsSkewBraceElm)
 BindGlobal("SkewBraceElmType", NewType(SkewBraceElmFamily, IsSkewBraceElm));
 BindGlobal("SkewBraceType", NewType(CollectionsFamily( SkewBraceElmFamily ), IsSkewBrace and IsAttributeStoringRep));
 
-# <#GAPDoc Label="SkewBrace">
-# <ManSection>
-#    <Oper Name="SkewBrace"
-#          Arg="t"/>
-#    <Returns>
-#          A skew brace
-#    </Returns>
-# <Description>
-# The argument <A>t</A> is a list of pairs of elements in a group.
-# <Example>
-# <![CDATA[
-# gap> SkewBrace([[(),()]]);
-# <brace of size 1>
-# gap> SkewBrace([[(),()],[(1,2),(1,2)]]);
-# <brace of size 2>
-# gap> SkewBrace( [ [ (), () ], [ (1,2)(3,4), (1,2)(3,4) ], 
-# > [ (1,3,2,4), (1,4,2,3) ], [ (1,4,2,3), (1,3,2,4) ] ] );
-# <brace of size 4>
-# ]]>
-# </Example>
-#
-# </Description>
-# </ManSection>
-# <#/GAPDoc>
+#!
+#! @Arguments t
+#! @Returns a skew brace 
+#! @Description
+#!  The argument <A>t</A> is a list of pairs of elements in a group.
+#! @ExampleSession
```

Comment:
This formatting is correct, and it should hopefully work with AutoDoc release made today. In the current version  2017.09.08 redistributed with GAP, there should be no space between `#!` and example.